src/base/utils.js

```diff
@@ -85,6 +85,14 @@ export var Fullscreen = {
     } else if(document.msExitFullscreen) {
       document.msExitFullscreen()
     }
+  },
+  canGoFullscreen: function() {
```

Comment:
I prefer using a standards based name, like `fullscreenEnabled` or `isFullscreenEnabled`. Why don't we go this way?